swc/private/swc.bzl

```diff
@@ -359,6 +364,22 @@ def _swc_impl(ctx):
                     output_sources.append(src)
                 continue
 
+            if _is_data_src(src_path):
+                # Copy data to the output directory.
+                # NOTE: assumes json must be resolved at runtime, see ts_project(resolve_json_module)
+                if ctx.attr.out_dir or True:
```

Comment:
`or True` is stray?

Reply:
Yeah that's somehow left form me hacking at something 🫣